templates/ai/what-is-kubeflow.html

```diff
@@ -8,9 +8,6 @@
 
 <section class="p-strip u-no-padding--top u-no-padding--bottom">
   <div class="p-strip--suru-topped">
-    {% if schedule_banner("2023-03-08", "2023-03-22") %}
-      {% include "shared/_charmed-kubeflow-beta-banner.html" %}
-    {% endif %}
```

Comment:
@carkod Can this be removed?

Reply:
I wouldn't remove it, we may need it soon in early 2024 again

Reply:
And why do you need to change this page? The ticket was for /pro/why-pro?